ggsdk/sdk.py

```diff
@@ -38,3 +40,16 @@ def save_dataset(self,
             exists (default=False)
         """
         self._core.save_dataset(data_generator, dataset_id, overwrite)
+
+    def get_data(self, module: str,
+                 profiles: typing.Union[str, typing.List[str]],
+                 revision: str):
+        """Call get data api.
+
+        :param module: Name of the module for the spring param path, e.g., nlp
+        :param profiles: List or string for the module's profile, e.g., test
+        :param revision: The revision for the associated param path, e.g., 2.0
+        """
+        return EnvironmentResponse(**json.loads(
+            self._core.get_data(module, profiles, revision).response.replace(
```

Comment:
I think to match our current pattern and make future refactor/simplification easier 
 the program flow should be slightly altered here. 
 
 I'm thinking that the `GraphGridSdk#get_data` method makes a one-line call to return `self._core.get_data`, and creation of `EnvironmentResponse`/processing of `response` would be pushed back into the in the `GetDataApi#handler` method.  That way we keep the `GraphGridSdk` clean. 
 
 
 Also I realize you're trying to keep match Spring which is why the class is named `EnvironmentResponse`. What would you think about renaming the class `GetDataResponse` instead, signifying the class represents the response from the `data` config call?  Open to others thoughts on this.

Reply:
I think all of these changes make sense. Done!